po/LINGUAS

```diff
@@ -0,0 +1 @@
+be ca da de el es fr gl it ja kk nl pt pt_BR ru tr uk zh_CN
```

Comment:
What's up about the i18n changes?  It looks like a partial replication of geany#2938, but without actually switching over to the "modernized" i18n setup I can't see where this is going.  What is this *actually* supposed to fix?  Again, explaining changes it very important :slightly_smiling_face:

Reply:
I reverted this change as I found that if I build from the root of the repo it works

Reply:
OK cool.  We can see to switch to the same technique as is used in Geany itself if it helps, but it's a tad more involved -- but IIUC that would be a good thing to avoid deprecated stuff.

Reply:
Two comments:
- we have #1183 which could make this better, even if not, it is a good change for itself (this is the pendant of geany#2938)
- I vagely remember some issue after the last or second last release when the translation files were not properly updated in the Windows build. Unfortunately, I didn't find any related issue or PR, neither in Geany nor in G-P :(.